internal/cluster/cluster_operations.go

```diff
@@ -401,7 +402,9 @@ func ImportCluster(ctx context.Context, data *schema.ResourceData, apiClient *cl
 	_ = data.Set("primary_datastore_type", clusterObj.PrimaryDatastoreType)
 	_ = data.Set("is_default", clusterObj.IsDefault)
 	_ = data.Set("is_stretched", clusterObj.IsStretched)
-	flattenedVdsSpecs := getFlattenedVdsSpecsForRefs(clusterObj.VdsSpecs)
+
+	flattenedVdsSpecs, err := getFlattenedVdsSpecsForRefs(ctx, clusterId, apiClient)
```

Comment:
```suggestion
	flattenedVdsSpecs, err := getFlattenedVdsSpecsForRefs(ctx, clusterId, apiClient)			 
    if err != nil {
    	return nil, err
	}
```

Reply:
Otherwise you will get an "ineffectual assignment to err (ineffassign)"

Reply:
code is in, i thought it was still there but i must have deleted it